examples/replication_dst.yaml

```
@@ -15,14 +15,8 @@ spec:
   deletion: false
   override: true
   enabled: true
-  dest_registry:
-    id: 1
-    name: test-registry
-    parentInstance:
-      name: test-harbor
-    type: docker-hub
-    url: https://hub.docker.com
-    insecure: false
+  dest_registry: # you have to create the dest_registry via a registry custom resource first
+    name: some-existing-registry
```

Nitpick: To keep the content of the examples folder with some integrity, maybe change this to the name of the example registry. This might avoid some confusion on the user-side:

https://github.com/mittwald/harbor-operator/blob/b9e45f4188f3ae6405bed2a501f57055c7891131/examples/registry.yaml#L4
```suggestion
    name: test-registry
```